style.css

```diff
@@ -0,0 +1,265 @@
+.body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+}
+
+.header::before {
+  content: '';
+  position: absolute;
+  width: 461px;
+  height: 1800px;
+  background: #000;
+}
+
+.header__title {
+  position: relative;
+  width: 98px;
+  height: 37px;
+  margin: 36px 0 0 52px;
+  font-family: Montserrat, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 30px;
+  line-height: 37px;
+  text-align: center;
+  letter-spacing: .07em;
+  color: #fff;
+}
+
+.header__language {
+  position: relative;
+  display: flex;
+  width: 238px;
+  justify-content: flex-end;
+  align-items: center;
+  top: 15px;
+}
+
+.header__language-ru {
+  position: relative;
+  width: 31px;
+  height: 25px;
+  font-family: Montserrat;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  color: #fff;
+  cursor: pointer;
+}
+
+.header__language-ru:: after {
+  content: "";
+  position: absolute;
+  background-image: url(./img/Vector.png);
+  height: 9px;
+  width: 13px;
+  left: 40px;
+  top: 9px;
+}
+
+.header__language-ru: hover {
+  transform: scale(1.3);
+ }
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Locc/pullRequest?prid=5986341)